main/lsp/ExtractVariable.cc

```diff
@@ -399,6 +399,12 @@ class ExpressionPtrSearchWalk {
             return;
         }
 
+        if (std::find(matches.begin(), matches.end(), tree.loc()) != matches.end()) {
```

Comment:
You can simplify this if you want:

```suggestion
        if (absl::c_find(matches, tree.loc()) != matches.end()) {
```

And in the newest version of abseil it simplifies even further to just `absl::c_contains(matches, tree.loc())`

https://github.com/abseil/abseil-cpp/commit/63d4b2fe1e0d4bd6a7b916f398643db40c35624b

I've opened a draft PR to try to see if we can trivially upgrade #8138 but do not block on this—just passing feedback.